internal/controller/controllers/clusterdeployments_controller_test.go

```diff
@@ -1090,8 +1103,272 @@ var _ = Describe("cluster reconcile", func() {
 			Expect(result).To(Equal(ctrl.Result{}))
 		})
 
+		It("Update manifests - skip cases", func() {
+			backEndCluster := &common.Cluster{
```

Comment:
isn't there a lot of common code here? maybe it should go into a context with a single before each that creates the cluster?